### ISSUE TITLE
Setting name is mentioned twice, once linked and once not

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#199389]
Testing #199247

Why is the setting name linked once and once it just shows the ID? Both should be linked, or maybe you can rephrase it to just use the setting name once like `Overwritten by Problems: Visibility when that setting is off`
<img width="548" alt="image" src="https://github.com/microsoft/vscode/assets/323878/f953ac04-9b10-4c76-9011-5eb4ffbce3a7">

**Comment by justschen** [comment]
That's a good point yeah
